.github/workflows/test.yaml

```diff
@@ -301,6 +301,7 @@ jobs:
       env:
           SPREAD_GOOGLE_KEY: ${{ secrets.SPREAD_GOOGLE_KEY }}
       run: |
+          set -o pipefail
```

Comment:
Thanks for this, I think I should be closer to the spread (i.e. one line further down) and a short comment why we need it. I also wonder if there is any alternative.

I personally consider `-o pipefail` an anti-pattern, consider:
```
$ cat /bin/snap|grep -m1 ELF; echo $?
Binary file (standard input) matches
0
$ set -o pipefail; cat /bin/snap|grep -m1 ELF; echo $?
Binary file (standard input) matches
141
```


Maybe something like:
``` 
exec &> >(tee "spread.log")
spread -abend google:${{ matrix.system }}:tests/... 
```
instead?

Reply:
Hmm how about I put the pipefail and spread into a subshell, so that we know if affects only that shell?

Reply:
Thank, I'm okay with that but would love to see the comment that Alberto suggested as well :)